resources/osl_mysql_test.rb

```diff
@@ -15,6 +15,8 @@
 
 # Install the mariadb package, set up the service, set up the user, then set up the given database
 action :create do
+  # Setup the port for mysql
+  osl_firewall_port 'mysql'
```

Comment:
I think this may be safer in the long run in case we run into issues with docker containers and such.
```suggestion
  osl_firewall_port 'mysql' do
    osl_only false
  end
```